plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -39,19 +38,22 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
   @VisibleForTesting(otherwise = PRIVATE)
   internal var isLocationComponentActivated = false
 
-  private val onIndicatorPositionChangedListener =
+  private val onIndicatorPositionChangedListeners =
     CopyOnWriteArrayList<OnIndicatorPositionChangedListener>()
 
-  private val onIndicatorBearingChangedListener =
+  private val onIndicatorBearingChangedListeners =
     CopyOnWriteArrayList<OnIndicatorBearingChangedListener>()
 
+  private val onIndicatorAccuracyRadiusChangedListeners =
```

Comment:
Any reasons on using `CopyOnWriteArrayList` instead of `CopyOnWriteArraySet`? 
Also same about listeners above

Reply:
Not sure, but `CopyOnWriteArraySet` is better, will update this.